src/main/java/cz/geokuk/core/coordinates/WgsParser.java

```diff
@@ -330,17 +340,24 @@ private Vzorek rozeber(final String s, final Matcher m, final int pozice) {
 			return null;
 		}
 
+		boolean vymenitSouradnice = false;
+
 		for (final PovolenaVariacePismen povapi : povoleneVariace) {
 			if (povapi.p1 == pismeno1 && povapi.p2 == pismeno2) {
 				return v; // je to OK
 			}
 			if (povapi.p1 == pismeno2 && povapi.p2 == pismeno1) {
-				final Souradky pom = v.sou1;
-				v.sou1 = v.sou2;
-				v.sou2 = pom;
-				return v; // je to OK, ale msueli jsme vyměnit strany
+				vymenitSouradnice = true;
```

Comment:
Ne, už vím, problém byl v sekvenci
		povolVariaci('-', ' ');
		povolVariaci(' ', '-');
Na tu druhou to nikdy nedošlo, ale místo toho původní kód přehodil souřadnice. Musí to tam být, když nebudu dělat větší zásah do kódu.